docs/network.md

```diff
@@ -0,0 +1,217 @@
+# 4. Code flow - finding first peer to connect to.
```

Comment:
Why does it start from 4?

Reply:
It starts at 1: https://github.com/near/nearcore/pull/5580